libs/Forc.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = [
+  "queue",
```

Comment:
I would move the queue to be in alphabetical order rather than at the top.

Reply:
Moved